website/docs/d/kms_key.html.markdown

```diff
@@ -31,20 +37,24 @@ resource "ibm_cos_bucket" "flex-us-south" {
 }
 ```
 
-**NOTE : Data of the key can be retrieved either using a key name or an alias name (if created for the key or keys) .
+**NOTE :
+1) Data of the key can be retrieved either using a key name or an alias name (if created for the key or keys) .
+2) limit is an optional parameter used with the keyname, which iterates and fetches the key till the limit given. When the limit is not passed then the first 2000 keys are fetched according to SDK default behaviour. 
```

Comment:
Is the default value 2000 or 200 in code we see it as 200

Reply:
Default value is 2000 if the user doesn't specify the limit but  if the limit is specified, the GetKey API fetches the Keys with a range of 200 keys per call